server/core/src/main/java/dev/slimevr/tracking/processor/skeleton/HumanSkeleton.kt

```diff
@@ -13,6 +13,7 @@ import dev.slimevr.tracking.trackers.TrackerRole
 import dev.slimevr.tracking.trackers.TrackerUtils.getFirstAvailableTracker
 import dev.slimevr.tracking.trackers.TrackerUtils.getTrackerForSkeleton
 import dev.slimevr.util.ann.VRServerThread
+import io.eiren.math.FloatMath.toRad
```

Comment:
Generally in the code we use `deg * FastMath.DEG_TO_RAD`, this method is probably fine to use tho, so idk what people's opinion on this is. This method is a lot shorter to call which I like.

Reply:
Addressed by switching to the constant.